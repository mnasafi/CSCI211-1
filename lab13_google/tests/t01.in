39.727565 -121.847693 211_Lab
39.729188 -121.846447 211_Lecture
39.729807 -121.842767 Celestinos_Pizza
39.729089 -121.842585 The_Bear
0
The_Bear
311_Lecture
Celestinos_Pizza
