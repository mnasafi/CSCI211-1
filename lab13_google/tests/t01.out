The_Bear is at http://maps.google.com/maps?q=39.7291+-121.843
311_Lecture not in database
Celestinos_Pizza is at http://maps.google.com/maps?q=39.7298+-121.843
